doc/user/01-create-first-user.md

```diff
@@ -0,0 +1,7 @@
+# Create the First User
+
+The first time a user will open the Atomic CRM App, they will be prompted to create the first user. The user will automatically be granted administrator role.
+
+./videos/01-create-first-user.webm
```

Comment:
Can't you embed this video using a real link?

![image](https://github.com/user-attachments/assets/a8b299e5-95eb-459c-b576-af2c299252b2)